src/CommonDBTM.php

```diff
@@ -6614,7 +6614,7 @@ public static function displayFullPageForItem(
 
         // Display extra html if needed
         if (!empty($options['after_display'] ?? "")) {
-            echo $options['after_display'];
+            echo htmlescape($options['after_display']);
```

Comment:
Where it is used, `after_display` contains the result of the `Ajax::createIframeModalWindow()` method that contains HTML code. It should not be encoded. Maybe the current method PHPDoc should focus on the fact that this option is expected to contain a safe HTML string.

```suggestion
            echo $options['after_display'];
```